go.mod

```diff
@@ -1,10 +1,13 @@
 module github.com/prebid/prebid-server/v3
 
-go 1.21
+go 1.22
```

Comment:
This should be extracted to a separate PR. We release Go upgrades separately to isolate potential issues introduced by the runtime.

Reply:
I see - ok then we will need to make a new one for that as prerequisite for this; will mark as draft until that time.

Reply:
- See #4102